js/src/modal.js

```diff
@@ -290,6 +294,31 @@ class Modal {
           this._element.focus()
         }
       })
+      .on(Event.KEYDOWN, (event) => {
+        if (event.isDefaultPrevented()) {
+          return
+        }
+        if (event.shiftKey && event.which === TAB_KEYCODE) {
+          const currentFocus = $(document.activeElement)
```

Comment:
No need to store it in a jQuery element.

```js
const currentFocus = document.activeElement
```

Reply:
Fixed.